inc/api/endpoints/controller/class-clonecomplete.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Pressbooks\Api\Endpoints\Controller;
+
+use Pressbooks\CloneTokens;
+
+class CloneComplete extends \WP_REST_Controller {
+
+	/**
+	 * @var string
+	 */
+	protected $namespace = 'pressbooks/v2/clone';
+
+	/**
+	 * @var string
+	 */
+	protected $rest_base = 'complete';
+
+	/**
+	 * @var CloneTokens
+	 */
+	protected $clone_tokens;
+
+	/**
+	 * @param CloneTokens $clone_tokens
+	 */
+	public function __construct( CloneTokens $clone_tokens ) {
+		$this->clone_tokens = $clone_tokens;
+	}
```

Comment:
Addressed. The `CloneComplete` was not injected, since it is not being used in some cases. It is used only in a particular case, so I think it is not a dependency, except for 1 method. I wrapped it up in a single line. Thanks for the suggestion!